admin/base/settings/defaults.py

```diff
@@ -102,8 +96,27 @@
     'addons.osfstorage',
     'addons.wiki',
     'addons.twofactor',
+
+    # Internal apps
+    'admin.common_auth',
+    'admin.base',
+    'admin.pre_reg',
+    'admin.spam',
+    'admin.metrics',
+    'admin.nodes',
+    'admin.users',
+    'admin.desk',
+    'admin.meetings',
+
 )
 
+MIGRATION_MODULES = {
```

Comment:
It is a Django setting: https://docs.djangoproject.com/en/1.10/ref/settings/#migration-modules .

> When you supply None as a value for an app, Django will consider the app as an app without migrations regardless of an existing migrations submodule. 

I also made `python manage.py migrate` crash if run from the admin server and used a custom router to make migrations in the admin project be noops. These are redundant, but I think the safeguards are worthwhile.